src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -15,6 +15,8 @@ import {
 } from '@box/metadata-editor';
 import noop from 'lodash/noop';
 
+import { FormValues } from '@box/metadata-editor/types/lib/components/metadata-instance-editor/types';
```

Comment:
Try to bump metadata-editor an import type directly from package

Reply:
nice